docs/package.json

```diff
@@ -5,8 +5,9 @@
   "license": "MIT",
   "scripts": {
     "dev": "next -p 8000",
-    "build": "next build",
+    "build": "next build && yarn sitemap",
     "start": "next start -p 8000",
+    "sitemap": "yarn next-sitemap --config sitemap.js",
```

Comment:
Generates `sitemap.xml` and `robots.txt` in the public directory after `next build` which are then accessible.